uniparc-rest/src/main/java/org/uniprot/api/uniparc/output/converter/UniParcXmlMessageConverter.java

```diff
@@ -24,7 +26,7 @@
     public UniParcXmlMessageConverter(String version) {
         super(UniParcEntry.class, UNIPARC_XML_CONTEXT);
         converter = new UniParcEntryConverter();
-        header = UNIPARC_XML_HEADER;
+        header = XML_DECLARATION + UNIPARC_XML_SCHEMA;
         if (Utils.notNullNotEmpty(version)) {
             String versionAttrib = " version=\"" + version + "\"" + ">\n";
             header = header.replace(">\n", versionAttrib);
```

Comment:
We have 2 closing tags for the header now.
I have a feeling that this code might replace twice, and this is not what we want, right?